index.html

```diff
@@ -0,0 +1,165 @@
+                    <li>2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="second_list">
+                    <button>html</button>
+                    <button>css</button>
+                    <button>javascript</button>
+                </ul>
+                <button class="see_project">See project</button>
+
+            </div>
+        </section>
+    </main>
+    <!-- footer -->
+    <footer>
+        <section class="section1">
+            <h1>About Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <span class="span">LET’S CONNECT</span>
+            <ul class="icons">
+                <i class='fa fa-twitter'></i>
+                <i class='fa fa-linkedin'></i>
+                <i class='fa fa-medium'></i>
+                <i class='fa fa-angellist'></i>
+                <i class='fa fa-github'></i>
+            </ul>
+            <button class="see_project">
+                Get my resume
+            </button>
+        </section>
+        <section class="section2">
+            <h1>
+                Languages
+            </h1>
+            <div class="lang">
+                <img src="/imgs/Ellipse 1 (2).png" alt=""></img>
+                <span>html</span>
+            </div>
+            <div class="lang">
+                <img src="/imgs/Ellipse 1 (1).png" alt=""></img>
+                <span>css</span>
+            </div>
+            <div class="lang">
+                <img src="/imgs/Ellipse 1.png" alt=""></img>
+                <span>javascript</span>
+            </div>
+        </section>
+        <section class="section3">
+            <div class="frameworks">
+                <span>Frameworks</span>
+                <span>&#10095;</span>
+            </div>
+            <div class="skills">
+                <span>Skills</span>
+                <span>&#10095;</span>
+            </div>
+        </section>
+    </footer>
+    <section class="form_section">
+        <h3>
+            Contact me
+        </h3>
+        <p>
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+I’d love to help with it!
+        </p>
+    <form action="https://formspree.io/f/mknalqrn" class="form" method="post">
+        
+        <input name="Email" id="email" type="email" required placeholder="Email">
+        <input name="name" id="name" type="text" required maxlength="30" placeholder="Name">
+        <input name="message" id="message" type="text" maxlength="500"  placeholder="Write your message here...">
+        <button type="submit" class="form_button">Get in touch</button>
+    </form>
+</section>
```

Comment:
- [ ] Kindly match your design to the Figma template. Here is the list of details you have to fix ⛏️ 
1 -  Please add more vertical spacing on the top of your heading **Contact me** 
2 - The width of all inputs should be identical 👯 
3 - Your button should be in the container 🥡 📦 
4 - At the bottom of your page, you should have a horizontal that matches the one in the Figma ____

Your design 👇🏽 👇🏽 

 ![Screenshot from 2023-06-07 08-55-18](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/10439283/ec49bac8-6462-4fad-8afc-d3ce6b49e1fd) 

Design from Figma 👇🏽 👇🏽 

 ![Screenshot from 2023-06-07 08-57-07](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/10439283/14a62948-096f-4ff0-9951-b15ebc082a61) |

Reply:
All of them are modified;
- [ ] 1 - Please add more vertical spacing on the top of your heading Contact me
- [ ] 2 - The width of all inputs should be identical 👯
- [ ] 3 - Your button should be in the container 🥡 📦
- [ ] 4 - At the bottom of your page, you should have a horizontal that matches the one in the Figma ____